RDBM/XML.h

```diff
@@ -0,0 +1,285 @@
+#ifndef XML_h_
+#define XML_h_
+#include"Debug\tinyxml.h"
+#include"Table.h"
+
+void XMLDecl()
+{
+	TiXmlDocument doc;
+	doc.LoadFile("dbdata.xml");
+	TiXmlDeclaration *decl = new TiXmlDeclaration("1.0", "UTF-8", "");
+	doc.LinkEndChild(decl);
+	TiXmlElement *list = new TiXmlElement("List");
+	doc.LinkEndChild(list);
+	doc.SaveFile("dbdata.xml");
+
+	TiXmlDocument doc2;
+	doc2.LoadFile("dbstructure.xml");
+	TiXmlDeclaration *decl2 = new TiXmlDeclaration("1.0", "UTF-8", "");
+	doc2.LinkEndChild(decl2);
+	TiXmlElement *list2 = new TiXmlElement("List");
+	doc2.LinkEndChild(list2);
+	doc2.SaveFile("dbstructure.xml");
+}
+
+bool check(Table& tb)
+{
+	TiXmlDocument doc;
+	doc.LoadFile("dbstructure.xml");
+	std::vector<std::string> names;
+	std::vector<char> types;
+
+	TiXmlElement *root = doc.RootElement();
+	if (NULL != root)
+	{
+		TiXmlElement *table = root->FirstChildElement("Table");
+		if (NULL != table)
+		{
+			while (table)
+			{
+				const char *at = table->Attribute("ID");
+				int at2 = tb.GetID();
+				std::string str = std::to_string(at2);
+				const char *at3 = str.c_str();
```

Comment:
just pass str.c_str() to the strcmp function. Please, use save functions for work with strings and add checking for str.length. because strcmp will say that strings 12 and 125 are equal.

Reply:
no ,it would not : in that case it would return -1